app/src/main/res/layout/fragment_subscriptions.xml

```diff
@@ -19,18 +19,31 @@
             app:title="@string/subscriptions_label"
             app:navigationIcon="?homeAsUpIndicator" />
 
-        <TextView
+        <LinearLayout
             android:id="@+id/feeds_filtered_message"
-            android:layout_width="match_parent"
+            android:layout_width="wrap_content"
```

Comment:
I would keep this at `match_parent` because it is a clickable view. I haven't tried running the PR but I think this will make the "glow" effect when clicking too short

Reply:
I changed it to `wrap_content` to reduce the clickable area to only the part around the icon and the text. Previously you could click on the right hand side of the screen and it would pickup the click which I thought would be bad UX.

I'm not sure what you mean by "glow" effect. Could you explain?

Reply:
I meant the ripple effect (background change) while clicking. I would expect to see that effect over the entire width of the screen. To me, it feels weird to see it stop at some point without some kind of visible "reason" to stop.

That being said, I now realize that the effect does not go all the way to the left either (even in the production version), which looks a bit strange too. I wonder if there was a reason to do that (touch target conflicts with the main menu?) or if it did not have a particular reason. One should probably have a look at `git blame` and the discussion in the PR that introduced the button.

Reply:
Cool. Thanks for explaining. I'll set it to `match_parent` so that existing functionality is preserved.